pubspec.lock

```diff
@@ -363,50 +363,50 @@ packages:
     dependency: "direct main"
     description:
       name: firebase_core
-      sha256: "3187f4f8e49968573fd7403011dca67ba95aae419bc0d8131500fae160d94f92"
+      sha256: "15d761b95dfa2906dfcc31b7fc6fe293188533d1a3ffe78389ba9e69bd7fdbde"
       url: "https://pub.dev"
     source: hosted
-    version: "3.3.0"
+    version: "3.9.0"
   firebase_core_platform_interface:
     dependency: transitive
     description:
       name: firebase_core_platform_interface
-      sha256: b94b217e3ad745e784960603d33d99471621ecca151c99c670869b76e50ad2a6
+      sha256: d7253d255ff10f85cfd2adaba9ac17bae878fa3ba577462451163bd9f1d1f0bf
       url: "https://pub.dev"
     source: hosted
-    version: "5.3.1"
+    version: "5.4.0"
   firebase_core_web:
     dependency: transitive
     description:
       name: firebase_core_web
-      sha256: "362e52457ed2b7b180964769c1e04d1e0ea0259fdf7025fdfedd019d4ae2bd88"
+      sha256: fbc008cf390d909b823763064b63afefe9f02d8afdb13eb3f485b871afee956b
       url: "https://pub.dev"
     source: hosted
-    version: "2.17.5"
+    version: "2.19.0"
   firebase_messaging:
     dependency: "direct main"
     description:
       name: firebase_messaging
-      sha256: "1b0a4f9ecbaf9007771bac152afad738ddfacc4b8431a7591c00829480d99553"
+      sha256: "151a3ee68736abf293aab66d1317ade53c88abe1db09c75a0460aebf7767bbdf"
       url: "https://pub.dev"
     source: hosted
-    version: "15.0.4"
+    version: "15.1.6"
   firebase_messaging_platform_interface:
```

Comment:
This should mean that all the Firebase packages are upgraded to latest:
      https://github.com/zulip/zulip-flutter/pull/1184#issuecomment-2555964673

You can confirm this yourself using `flutter pub outdated`. Here's the output I see at the tip of the branch:
```
$ flutter pub outdated
Showing outdated packages.
[*] indicates versions that are not the latest available.

Package Name              Current              Upgradable           Resolvable           Latest   

direct dependencies:     
collection                *1.19.0              *1.19.0              *1.19.0              1.19.1   
device_info_plus          *10.1.2              *10.1.2              11.2.0               11.2.0   
flutter_color_models      *1.4.0 (overridden)  *1.4.0 (overridden)  *1.4.0 (overridden)  1.3.3+2  
intl                      *0.19.0              *0.19.0              *0.19.0              0.20.1   
mime                      *1.0.6               *1.0.6               2.0.0                2.0.0    
wakelock_plus             *1.2.8               1.2.9                1.2.9                1.2.9    

dev_dependencies:        
flutter_lints             *4.0.0               *4.0.0               5.0.0                5.0.0    
json_serializable         *6.9.0               *6.9.0               *6.9.0               6.9.2    
pigeon                    *20.0.2              *20.0.2              22.7.0               22.7.0   
stack_trace               *1.12.0              *1.12.0              *1.12.0              1.12.1   

transitive dependencies: 
characters                *1.3.0               *1.3.0               *1.3.0               1.4.0    
matcher                   *0.12.16+1           *0.12.16+1           *0.12.16+1           0.12.17  
material_color_utilities  *0.11.1              *0.11.1              *0.11.1              0.12.0   
meta                      *1.15.0              *1.15.0              *1.15.0              1.16.0   
stream_channel            *2.1.2               *2.1.2               *2.1.2               2.1.3    
string_scanner            *1.4.0               *1.4.0               *1.4.0               1.4.1    
term_glyph                *1.2.1               *1.2.1               *1.2.1               1.2.2    
vm_service                *14.3.1              *14.3.1              *14.3.1              15.0.0   
win32_registry            *1.1.5               *1.1.5               *1.1.5               2.0.1    

transitive dev_dependencies:
_fe_analyzer_shared       *76.0.0              *76.0.0              *76.0.0              78.0.0   
analyzer                  *6.11.0              *6.11.0              *6.11.0              7.1.0    
analyzer_plugin           *0.11.3              *0.11.3              *0.11.3              0.12.0   
dart_style                *2.3.7               *2.3.7               *2.3.7               3.0.1    
lints                     *4.0.0               *4.0.0               5.1.1                5.1.1    
pubspec_parse             *1.3.0               1.4.0                1.4.0                1.4.0    
source_gen                *1.5.0               *1.5.0               *1.5.0               2.0.0    
yaml                      *3.1.2               3.1.3                3.1.3                3.1.3    

3 upgradable dependencies are locked (in pubspec.lock) to older versions.
To update these dependencies, use `flutter pub upgrade`.

5  dependencies are constrained to versions that are older than a resolvable version.
To update these dependencies, edit pubspec.yaml, or run `flutter pub upgrade --major-versions`.
```

I think most of those are probably from the Flutter version being a couple of weeks old. In any case, no Firebase packages in the list.

In the terminal it comes in helpful color, too. Here's a screenshot:
![image](https://github.com/user-attachments/assets/3545953b-d8c7-45a0-b4b7-f6b6ab1d236a)